### PR TITLE
Feat: Added list to the CL

### PR DESCRIPTION
**Pull request by zoef-yt**
Added lists to CL

### CODE REVIEW

**Review by tonymike99 — APPROVED** [review]
Neat code. Approved.

File: pages/components/list/list.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+	display: flex;
+	flex-direction: column;
+	width: 100%;
+	transition: all 0.2s ease;
+}
+
+.list-group-item {
+	position: relative;
+	display: block;
+	padding: 0.5rem 1rem;
```
Comment (Omkar-Ghate) [inline pages/components/list/list.css]: maybe you could use px here?

**Review by Omkar-Ghate — APPROVED** [review]
Great work. Looks good to merge.

**Comment by zoef-yt** [comment]
Thanks for the review.

**Pull request merged**